ui/src/AppCharting/AppChartC3.svelte

```diff
@@ -130,9 +133,8 @@
               dateTime,
               size
             };
-            let error_list = errors.get(id);
-            if (error_list) {
-              error_list.push(new_error);
+            if (errors.has(id)) {
```

Comment:
For my own edification, is this preferable because it avoids making a copy? Or is it just one fewer variable / reference to worry about?

Reply:
Yeah, though it would still be a pointer so probably not that expensive, but I just like using it now that we have it on `Map<>`